src/main/kubectl/kubectl.ts

```diff
@@ -299,7 +299,7 @@ export class Kubectl {
       "elif test -f \"$HOME/.profile\"; then",
       "  . \"$HOME/.profile\"",
       "fi",
-      `export PATH="${binariesDir}:${kubectlPath}:$PATH"`,
+      `export PATH="${kubectlPath}:${binariesDir}:$PATH"`,
```

Comment:
these changes don't help on Windows (it still uses the bundled kubectl)

Reply:
Fixed